spec/requests/user_status_controller_spec.rb

```diff
@@ -75,6 +75,19 @@
         expect(response.status).to eq(400)
       end
 
+      it "validates emoji" do
+        put "/user-status.json", params: { description: "girl_with_pink_hair" }
+        expect(response.status).to eq(400)
+      end
+
+      it "limits description parameter to 100 characters" do
```

Comment:
```suggestion
      it "limits description’s length" do
```

We don't really need to know about 100 here, and if we change it, there's high chances we forget to update this.